src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import Navbar from './components/Navbar';
+import './App.css';
+
+const App = () => (
+  <div className="App">
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route exact path="/" element={<Books />} />
+        <Route path="bookstore" element={<Books />} />
```

Comment:
- [ ] Please, consider that the previous review request to remove line 14, because you have exact part for Books already 👍